### ISSUE TITLE
SST Issue with Load Balancer when deploying cluster with multiple containers

### ISSUE THREAD

**Issue by AdityaBanik** [anomalyco/sst#5078]
Lets say I want to deploy 2 containers and i want a custom domain.
I only have option to forward ports in load balancer with assigned containers as target groups.

`        ports: [
          { listen: '8000/http', container: 'backend' },
          {
            listen: '443/https',
            forward: '8000/http',
            container: 'backend',
          },
          { listen: '4000/http', container: 'graphql' },`
          
Now lets say i want to do something like https://www.example.com/graphql to listen on port 443/https and do path based forwarding to port 4000 through load balancer i cant. I have to go to aws load balancer to set rules manually.

I wish there was an easy way to do this.

**Comment by fwang** [comment]
Released in v3.3.37.

```ts
loadBalancer: {
  ports: [
    { listen: '8000/http', container: 'backend' },
    { listen: '8000/http', container: 'graphql', path: '/graphql/*' }
  ],
},
```

**Comment by AdityaBanik** [comment]
Hey @fwang  @jayair  ❤️Thank you for adding this feature. Really appreciate the community efforts to make sst great.
